src/App.js

```diff
@@ -1,28 +1,27 @@
-import { Routes, Route } from 'react-router-dom';
-import './App.css';
-import UserList from './components/UserList';
-import UserDetails from './components/UserDetails';
-import NavBar from './components/NavBar';
-import { Component } from 'react';
-import UserEdit from './components/UserEdit';
-import UserCreation from './components/UserCreation';
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import "./App.css";
+import UserList from "./components/UserList";
+import UserDetails from "./components/UserDetails";
+import NavBar from "./components/NavBar";
+import { Component } from "react";
+import UserEdit from "./components/UserEdit";
+import UserCreation from "./components/UserCreation";
 
 class App extends Component {
-
   render() {
     return (
-      <>
+      <Router>
         <NavBar />
         <div className="App p-10 bg-slate-100">
-          <Routes>
-            <Route path='/users' element={<UserList />}></Route>
-            <Route path='/users/details/:userId' element={<UserDetails />}></Route>
-            <Route path='/users/edit/:userId' element={<UserEdit />}></Route>
-            <Route path='/users/new' element={<UserCreation />}></Route>
-          </Routes>
+          <Switch>
```

Comment:
In React router V6 and `<Switch>` replaced by `<Routes>`.
More details [here](https://stackoverflow.com/questions/70662160/switch-or-routes-which-is-it-with-react-router)